src/vs/workbench/api/common/extHostTerminalService.ts

```diff
@@ -867,8 +871,9 @@ export abstract class BaseExtHostTerminalService extends Disposable implements I
 	}
 }
 
-class EnvironmentVariableCollection implements vscode.EnvironmentVariableCollection {
+class EnvironmentVariableCollection {
```

Comment:
We should be able to keep this?

Reply:
No, as this is not what is exposed to the user, see https://github.com/microsoft/vscode/pull/182238#discussion_r1192860115.